nom-derive-impl/src/config.rs

```diff
@@ -1,10 +1,11 @@
 use crate::endian::ParserEndianness;
 use crate::meta::attr::{MetaAttr, MetaAttrType};
 use proc_macro2::{Span, TokenStream};
-use syn::{spanned::Spanned, Error};
+use syn::{Error, spanned::Spanned};
 
 #[derive(Debug)]
 pub struct Config {
+    #[allow(dead_code)]
```

Comment:
This is ok for a temporary workaround, but I think we should remove the unused fields (or use them)